tests/test_date.py

```diff
@@ -284,6 +284,16 @@ def test_should_parse_date_with_timezones_using_format(self):
         self.assertEqual('day', date_data['period'])
         self.assertEqual(expected, date_data['date_obj'])
 
+    @parameterized.expand([
+        param(['ur', 'li']),
+        param(['pk',]),
+    ])
+    def test_should_raise_error_when_unknown_language_given(self, shortnames):
+        with self.assertRaises(ValueError) as context:
+            date.DateDataParser(languages=shortnames)
+
+        self.assertTrue(context.exception)
```

Comment:
Thank you for pointing this out. Fixed it and modified `assertRaise` to `assertRaiseRegexp` to be better.